picard/metadata.py

```diff
@@ -153,6 +156,15 @@ class Metadata(MutableMapping):
         ('totaldiscs', 4),
     ]
 
+    __date_match_factors = dict([
```

Comment:
Please use normal dict declaration - 

```
VALUES = {
    "key": "value",
    ...
}

Reply:
I made the change as requested, and I have no concern with this,  but what I don't understand, is that just one line above, the pre-existing syntax used for __weights uses the syntax I had.  so now just one line apart we have two ways of doing the same thing.  I would think consistency would be more important here.

Reply:
> the pre-existing syntax used for __weights

`__weights` is a list while `__date_match_factors` is a dict.

That said, the syntax you used is perfectly acceptable, but you create a dictionnary by passing a list of tuples to constructor. Using dict literal syntax `{}` is known as being faster (even though the time difference was significantly reduced with Python3 and even more with recent 3.x versions).

See: https://stackoverflow.com/questions/6610606/is-there-a-difference-between-using-a-dict-literal-and-a-dict-constructor

Reply:
I can see the new syntax is better, I just wasn't getting the difference from __weights.  but in tracing through the code, I can see weights is used differently in this class.  Even differently than the `comparison_weights` are used in file.py. which actually is using the same syntax you are suggesting here :)